script.js

```diff
@@ -0,0 +1,13 @@
+/* eslint-disable no-unused-vars */
```

Comment:
- [ ] I noticed that you disable a lot **Eslint** . Please remove the **eslint disable** statement and try to fix all the errors you are getting. Because ` Eslint` helps developers to write clean JavaScript codes and adapt best practices when writing the code.  :+1:

Reply:
Added ESlint to make my code clean